src/Geolocation.svelte

```diff
@@ -43,10 +46,12 @@
   /** `true` if the browser does not support the Geolocation API */
   export let notSupported = false;
 
-  import { createEventDispatcher } from "svelte";
+  import { createEventDispatcher, onDestroy } from 'svelte'
```

Comment:
```suggestion
  import { createEventDispatcher, onDestroy } from "svelte";
```

Double quotes and a semicolon